README.md

```diff
@@ -19,97 +20,173 @@ This WordPress project is intended to provide:
 - Homebrew (1.2.3)
   - MariaDB (10.0 or greater)
   - Wordpress Command Line (1.2.1)
+  - Composer (1.4.2)
 - PHP 7
+- Codeception for Wordpress (Wp-Browser 1.21)
+
+***
 
 ## Installation
 
-  Installation process for the setting up Wordpress on a local environment.
+Installation process for the setting up Wordpress on a local environment.
+
+- **Create the MySQL user and database using the MariaDB monitor**.
+
+```
+mysql -uroot
+CREATE DATABASE chappress_dev;
+GRANT ALL PRIVILEGES ON chappress_dev.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
+FLUSH PRIVILEGES;
+EXIT
+```
+
+- **Create A Testing Database used for Codeception**.
+
+```
+mysql -uroot
+CREATE DATABASE wpTests;
+GRANT ALL PRIVILEGES ON wpTests.* TO "wpTests"@"localhost" IDENTIFIED BY "wpTests";
+FLUSH PRIVILEGES;
+EXIT
+```
```

Comment:
I'd like this changed, I don't like camelCase and in the previous example were using underscore_var_names. *Unless* you can show that camel-case is recommended PHP practice. (I'm not sure.)

In any case, let's the same user as for the dev database. Just add steps above to create a `chappress_test` database and give `chappress` user access to that.

Then update test config to use that database.